### PR TITLE
Use stylelint to enforce selector-list-comma-newline-after #7096

### LINKED ISSUE #7095: CSS files contain instances of multiple selectors written in the same line
According to our CSS [coding standard](https://docs.google.com/document/d/1wA9paRA9cS7ByStGbhRRUZLEzEzimrNQjIDPVqy1ScI/pub), selectors must be specified in separate lines.

However, there are several violations of this rule in `teammatesCommon.css`, `datepicker.css` and `omniComment.css`.

### LINKED ISSUE #7096: Use stylelint to enforce selector-list-comma-newline-after
The [rule](https://stylelint.io/user-guide/rules/selector-list-comma-newline-after/) can help to prevent issues like #7095.

### PR DESCRIPTION
**Pull request by sujeet14108**
Fixes #7096, fixes #7095 
Added rule 
selector-list-comma-newline-after: always

screenshot from 2017-04-13 00-16-42

### CODE REVIEW

File: static-analysis/teammates-stylelint.yml [hunk 1]
```diff
@@ -530,3 +530,6 @@ rules:
 
     # Disallow animation names that do not correspond to a @keyframes declaration
     # no-unknown-animations
+
+    #Require a newline or disallow whitespace after the commas of selector lists.
```
Comment (whipermr5) [inline static-analysis/teammates-stylelint.yml]: Can this be `# One selector per line.`? Makes the intention clearer.

**Comment by sujeet14108** [comment]
@whipermr5 Done :)

File: static-analysis/teammates-stylelint.yml [hunk 2]
```diff
@@ -530,3 +530,6 @@ rules:
 
     # Disallow animation names that do not correspond to a @keyframes declaration
     # no-unknown-animations
+
+    # One selector per line.
+    selector-list-comma-newline-after: always
```
Comment (wkurniawan07) [inline static-analysis/teammates-stylelint.yml]: You should not actually need this. Somewhere in this YML file there should already be this rule, but commented out -- you only need to un-comment it.

Comment (whipermr5) [inline static-analysis/teammates-stylelint.yml]: Oops didn't notice 😅

**Comment by sujeet14108** [comment]
@whipermr5 check now :)

File: static-analysis/teammates-stylelint.yml [hunk 3]
```diff
@@ -530,3 +531,4 @@ rules:
 
     # Disallow animation names that do not correspond to a @keyframes declaration
     # no-unknown-animations
+
```
Comment (whipermr5) [inline static-analysis/teammates-stylelint.yml]: This newline is an unrelated change.

File: static-analysis/teammates-stylelint.yml [hunk 4]
```diff
@@ -383,7 +383,8 @@ rules:
     # Selector list
 
     # Require a newline or disallow whitespace after the commas of selector lists
-    # selector-list-comma-newline-after
+    # One selector per line.
```
Comment (whipermr5) [inline static-analysis/teammates-stylelint.yml]: Now that this comment is here under another comment, it might be better to provide more context e.g. `Used to enforce one selector per line.`

**Comment by sujeet14108** [comment]
@whipermr5  Check now :)
